Composer/packages/server/src/controllers/asset.ts

```diff
@@ -2,12 +2,14 @@
 // Licensed under the MIT License.
 
 import { Request, Response } from 'express';
-import { BotTemplateV2, QnABotTemplateId } from '@bfc/shared';
+import { BotTemplate, QnABotTemplateId } from '@bfc/shared';
 import formatMessage from 'format-message';
 
 import AssetService from '../services/asset';
 import { getNpmTemplates } from '../utility/npm';
 import log from '../logger';
+import { templateSortOrder } from '../constants';
+import _ from 'lodash';
```

Comment:
Import just the funcs you need from lodash. Not really a problem on the server, but its best to remain consistent with client & server.

```ts
import union from 'lodash/union';
```

Reply:
Handled